### PR TITLE
Backup codes API definitions modifications

### PR DESCRIPTION
**Pull request by lashinijay**
## Related Git Issue
- [https://github.com/wso2-enterprise/asgardeo-product/issues/11121](https://github.com/wso2-enterprise/asgardeo-product/issues/11121)

## Related PRs
- [https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6](https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6)

### CODE REVIEW

File: components/org.wso2.carbon.identity.api.user.backupcode/org.wso2.carbon.identity.api.user.backupcode.v1/src/main/resources/backupcode.yaml [hunk 1]
```diff
@@ -217,3 +195,5 @@ securityDefinitions:
 # Added by API Auto Mocking Plugin
 host: localhost:9443
 basePath: /t/{tenant-domain}/api/users/v1
+
+
```
Comment (somindatommy) [inline components/org.wso2.carbon.identity.api.user.backupcode/org.wso2.carbon.identity.api.user.backupcode.v1/src/main/resources/backupcode.yaml:199]: Remove unwanted newlines

Comment (lashinijay) [reply components/org.wso2.carbon.identity.api.user.backupcode/org.wso2.carbon.identity.api.user.backupcode.v1/src/main/resources/backupcode.yaml]: addressed with [https://github.com/wso2/identity-api-user/pull/150/commits/85df13f8207a3773af36a2450976fd8adaac53cf](https://github.com/wso2/identity-api-user/pull/150/commits/85df13f8207a3773af36a2450976fd8adaac53cf)

File: components/org.wso2.carbon.identity.api.user.backupcode/org.wso2.carbon.identity.api.user.backupcode.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/backupcode/v1/core/BackupCodeService.java [hunk 2]
```diff
@@ -87,8 +88,8 @@ public BackupCodeResponseDTO initBackupCodes() {
         BackupCodeResponseDTO backupCodeResponseDTO = new BackupCodeResponseDTO();
         User user = getUser();
```
[the remark this answers is not in the document]
Comment (lashinijay) [reply components/org.wso2.carbon.identity.api.user.backupcode/org.wso2.carbon.identity.api.user.backupcode.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/backupcode/v1/core/BackupCodeService.java]: addressed with [https://github.com/wso2/identity-api-user/pull/150/commits/85df13f8207a3773af36a2450976fd8adaac53cf](https://github.com/wso2/identity-api-user/pull/150/commits/85df13f8207a3773af36a2450976fd8adaac53cf)

File: components/org.wso2.carbon.identity.api.user.backupcode/org.wso2.carbon.identity.api.user.backupcode.v1/src/main/resources/backupcode.yaml [hunk 3]
```diff
@@ -216,4 +194,4 @@ securityDefinitions:
     tokenUrl: https://localhost:9443/oauth2/token
 # Added by API Auto Mocking Plugin
 host: localhost:9443
-basePath: /t/{tenant-domain}/api/users/v1
+basePath: /t/{tenant-domain}/api/users/v1
```
Comment (somindatommy) [inline components/org.wso2.carbon.identity.api.user.backupcode/org.wso2.carbon.identity.api.user.backupcode.v1/src/main/resources/backupcode.yaml:197]: Add a new line here

**Review by somindatommy — APPROVED** [review]

**Pull request merged**